lms/templates/seq_module.html

```diff
@@ -3,6 +3,18 @@
 
 <div id="sequence_${element_id}" class="sequence" data-id="${item_id}" data-position="${position}" data-ajax-url="${ajax_url}" data-next-url="${next_url}" data-prev-url="${prev_url}">
   <div class="path"></div>
+    % if masquerade_override:
+    <div class="pattern-library-shim">
+      <div class="alert alert-information subsection-header" role="alert" tabindex="-1">
```

Comment:
I think it would be cleaner if the ```.pattern-library-shim``` was on the same element as the ```.alert``` class, so when we do move to the pattern library we are just removing a class instead of a wrapper DOM element.